native/KotlinIOS/KotlinIOS/customTableViewCell.swift

```diff
@@ -21,4 +23,16 @@ class customTableViewCell: UITableViewCell {
         // Configure the view for the selected state
     }
     
+    func setJourney(journey:JourneyTableDataElem){
+        departureStation.text = journey.startStation.shortName
```

Comment:
Could we pass through just what we need into this function i.e. change the object passed into the function? As in to avoid us having to call startStation.shortName if shortName is the only part we use, and also so we don't have to call e.g. getStartTime() here.